src/components/ReleaseMonitor/ReleaseEmptyState.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { EmptyStateBody, Text, TextVariants } from '@patternfly/react-core';
+import emptyStateImgUrl from '../../assets/Pipeline.svg';
+import AppEmptyState from '../../shared/components/empty-state/AppEmptyState';
+
+const MonitoredReleaseEmptyState: React.FC<React.PropsWithChildren<unknown>> = () => (
+  <AppEmptyState emptyStateImg={emptyStateImgUrl} title="No releases found">
+    <EmptyStateBody>
+      <Text component={TextVariants.p}>
+        You don&apos;t have available releases matching the current filters. Please try adjusting
```

Comment:
it is quite complex here. If you check other pages without resource, the empty state is much cleaner. would you like to make the explanation more concise?